src/EditorFeatures/Core/GoToImplementation/GoToImplementationCommandHandler.cs

```diff
@@ -14,12 +13,14 @@
 using Microsoft.CodeAnalysis.Shared.Extensions;
 using Microsoft.CodeAnalysis.Text;
 using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.UI.Commanding;
+using Microsoft.VisualStudio.Text.UI.Commanding.Commands;
 
 namespace Microsoft.CodeAnalysis.Editor.GoToImplementation
 {
-    [ExportCommandHandler(PredefinedCommandHandlerNames.GoToImplementation,
+    [ExportLegacyCommandHandler(PredefinedCommandHandlerNames.GoToImplementation,
```

Comment:
Why did this one stay legacy? Since it's our own command, we don't yet have a way to dispatch it?

Reply:
Correct--editor doesn't dispatch this one yet.